crates/holochain/src/core/ribosome/host_fn/zome_info.rs

```diff
@@ -71,10 +71,10 @@ pub mod test {
         assert_eq!(
             zome_info.extern_fns,
             vec![
-                FunctionName::new("__allocate"),
                 FunctionName::new("__data_end"),
-                FunctionName::new("__deallocate"),
                 FunctionName::new("__getrandom_custom"),
+                FunctionName::new("__hc__allocate_1"),
+                FunctionName::new("__hc__deallocate_1"),
```

Comment:
@maackle the others are externs either from the happ or callbacks or wasmer

Reply:
Could you spell that out a bit more? Why *don't* those things you mentioned get versions? And are alloc and dealloc best considered hdk functions? What category are they in?

Just seems weird that there would be an exception in this list and I want to understand it.

Reply:
@maackle - these are all the symbol exports in the C linking sense. The wasmer ones we have no control of... they are exported by wasmer. The happ externs are not prefixed... there's no mangling needed there because the happ dev is naming them, there's nothing to collide with.